coalib/parsing/FilterHelper.py

```diff
@@ -75,8 +76,11 @@ def apply_filters(filters, bears=None, sections=None):
         items = sections
         applier = _apply_section_filter
 
-    for filter in filters:
-        filter_name, *filter_args = filter
+    for filter_name, *filter_args in filters.items():
+        for filter_arg in filter_args:
+            if isinstance(filter_arg, tuple):
```

Comment:
How is this code still relevant? Can't `filter_vector_to_dict` take care of all this?

Reply:
I thought filter_vector_to_dict was supposed to change the input to a dict. And this code is supposed to parse through that input.